src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Edited the documentation for the cmdlet `Set-AzVMADDomainExtension` to ensure the example is accurate. 
```

Comment:
I think we don't need to publish new version for a doc improvement. What's your thought? Could you please remove this change log?

Reply:
Updated docs will be available to customer in April 5th. No matter a new version of module whether publishes.

Reply:
I'll remove the changelog update.